spring-websockets/src/main/java/com/baeldung/debugwebsockets/StompWebSocketClient.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.debugwebsockets;
+
+import org.springframework.messaging.converter.MappingJackson2MessageConverter;
+import org.springframework.messaging.simp.stomp.StompSessionHandler;
+import org.springframework.web.socket.client.WebSocketClient;
+import org.springframework.web.socket.client.standard.StandardWebSocketClient;
+import org.springframework.web.socket.messaging.WebSocketStompClient;
+
+import java.util.Scanner;
+
+public class StompWebSocketClient {
+
+    private static final String URL = "ws://localhost:8080/stock-ticks/websocket";
+
+    public static void main(String[] args) {
+
+        WebSocketClient client = new StandardWebSocketClient();
+        WebSocketStompClient stompClient = new WebSocketStompClient(client);
+        stompClient.setMessageConverter(new MappingJackson2MessageConverter());
+        StompSessionHandler sessionHandler = new StompClientSessionHandler();
+        stompClient.connect(URL, sessionHandler);
+
+        new Scanner(System.in).nextLine();
+
```

Comment:
As per guidelines, let's remove the empty lines at the beginning and end of this block

Reply:
Thank You @KevinGilmore.  Made the change